src/BaGet.GCP/Configuration/GoogleCloudStorageOptions.cs

```diff
@@ -0,0 +1,9 @@
+using BaGet.Core.Configuration;
+
+namespace BaGet.GCP.Configuration
+{
+    public class GoogleCloudStorageOptions : StorageOptions
+    {
+        public string BucketName { get; set; }
```

Comment:
Please mark this as `[Required]`. For example, see: https://github.com/loic-sharma/BaGet/blob/master/src/BaGet.Azure/Configuration/BlobStorageOptions.cs#L7